axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/PurchaseOrderInvoiceServiceImpl.java

```diff
@@ -425,4 +441,75 @@ public PurchaseOrder fillPurchaseOrder(PurchaseOrder purchaseOrder, Invoice invo
 
     return purchaseOrder;
   }
+
+  @Override
+  @Transactional(rollbackOn = Exception.class)
+  public Invoice mergeInvoice(
+      List<Invoice> invoiceList,
+      Company company,
+      Currency currency,
+      Partner partner,
+      Partner contactPartner,
+      PriceList priceList,
+      PaymentMode paymentMode,
+      PaymentCondition paymentCondition,
+      String supplierInvoiceNb,
+      LocalDate originDate,
+      PurchaseOrder purchaseOrder)
+      throws AxelorException {
+    log.debug("service supplychain 1 (purchaseOrder) {}", purchaseOrder);
```

Comment:
not sure about this log message, please remove it or make the message more explicit